packages/devextreme-react/src/core/options-manager.ts

```diff
@@ -94,6 +94,18 @@ class OptionsManager {
     this.isUpdating = false;
     this.instance.endUpdate();
 
+    const options = this.instance.option();
+    if (options !== null && options !== undefined) {
+      Object.keys(config.options).forEach((propName) => {
+        if (config.options[propName] !== options[propName]) {
+          this.setValue(
+              propName,
+              this.wrapOptionValue(propName, config.options[propName]),
+          );
```

Comment:
The `setValue` already uses `wrapOptionValue` in `instance.option` method.
I think `wrapOptionValue` usage is excessive, if we use `setValue`. Or we should use `this.instance.option` instead `setValue`